AWS-CodePipeline/sam-monorepo-pipeline-template/{{cookiecutter.outputDir}}/samconfig.toml

```diff
@@ -0,0 +1,34 @@
+version = 0.1
```

Comment:
What if customers already have a samconfig.toml in their project folder? Is there a way to do it without creating this file? (like providing CLI option in codebuild project, or use another file name)

Reply:
Good point. I've decided to instead put this `samconfig.toml` inside the `pipeline` folder (together with the buildspecs, etc), and reference it in calls to `sam` using `--config-file`.
This way it still gives the customer the opportunity to _centrally_ manage particularities to each build/deploy cycle (e.g. `parameter_overrides`) without interfering with the _default_ `samconfig.toml`.

I'm validating this new approach, and will update this PR shortly.

Reply:
Solved. `samconfig.toml` moved to `pipeline/` folder, and buildspecs updated accordingly.

Reply:
thanks!